aimm_adapters/labview_reader.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+
+
+class LabviewFileReader:
+
+    structure_family = "dataframe"
+
+    def __init__(self, path):
+        self._file = open(path, "r")
+
+    def parse_file(self):
```

Comment:
I think it would make sense to move this out into a _function_ so it can be tested (and perhaps resused) separately from the Tiled API here, which will grow methods like `macrostructure`, `microstructure`, `read`....